lib/main.coffee

```diff
@@ -1,11 +1,30 @@
 {CompositeDisposable} = require('atom')
 
+# The default grammar scopes checked by this linter.
+defaultGrammarScopes = [
+  'text.html.basic'
+]
+
 module.exports =
   config:
     executablePath:
       default: 'tidy'
       title: 'Full path to the `tidy` executable'
       type: 'string'
+    customGrammarScopes:
+      default: []
```

Comment:
Put `text.html.basic` in as the default here, that way users can see what it already is doing, and also allows them to remove it if they want to for some reason.